Covid-19 Real Time notification (#969)
avinashkranjan/Amazing-Python-Scripts: Covid-19 _Real-time_Notification/readME.md

# REAL TIME COVID-19 OUTBREAK NOTIFICATION

> This is an notifier built with the use of BeautifulSoup, which will keep updating the user about the situation of covid in a particular state the user wants to know about after a particular interval of time.

Reviewer: ```suggestion
This is a notifier built with the use of BeautifulSoup, which will keep updating the user about the situation of covid in a particular state the user wants to know about after a particular interval of time.
```

# What is the use ?

> In this time of crisis it is needed to keep ourselves updated with the cases of covid for our own safety and benefit, and to get a better view of the situation.

Reviewer: ```suggestion
In this time of crisis, it is needed to keep ourselves updated with the cases of covid for our own safety and benefit, and to get a better view of the situation.
```

> Everyone would like to have system, that keeps them updated about the total cases, cured and deaths at an interval on its own. This script does exactly that for the user!

Reviewer: ```suggestion
Everyone would like to have a system, that keeps them updated about the total cases, cured, and deaths at an interval on its own. This script does exactly that for the user!
```

# How to Use :

Once the user starts the program, it will ask you about the state whose data you want to know, after it is entered, the notification system will do the work by itself and keep updating the user after an interval of an hour. The script will keep running in the background until the user exits it manually.

# Modules used :

Plyer
requests
bs4
time

# Development Status

> This script is at present complete. A future version may include display of data od various states and also compare the new cases to cured cases , give update on the vaccine distribution etc.

Reviewer: ```suggestion
This script is at present complete. A future version may include a display of data of various states and also compare the new cases to cured cases, give updates on the vaccine distribution, etc.
```

> # Developed by  Sayantani Saha

Reviewer: ```suggestion
## Developed by  [Sayantani Saha](https://github.com/sayantani11)
```

> # Developed by  [Sayantani Saha]  (https://github.com/sayantani11)

Reviewer: ```suggestion
# Developed by  [Sayantani Saha](https://github.com/sayantani11)
```
